### PR TITLE
1.3 Metrics - Adding Callout For Ephemeral Models

### PR DESCRIPTION
**Pull request by callum-mcdata**
## Description & motivation
Adding a small admonition to ensure people know that metrics can't be built on ephemeral models

### CODE REVIEW

File: website/docs/docs/building-a-dbt-project/metrics.md [hunk 1]
```diff
@@ -135,6 +135,12 @@ metrics:
 
 </File>   
 
+:::caution
+
+- Metrics cannot be built on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). The materialization must have a represention in the data warehouse in order to build a metric.
```
Comment (mirnawong1) [inline website/docs/docs/building-a-dbt-project/metrics.md]: switch to active voice and fix representation typo

```suggestion
- You cannot build metrics on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). To build a metric, the materialization must have a representation in the data warehouse.
```

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
small tweak and good to merge @callum-mcdata !

File: website/docs/docs/building-a-dbt-project/metrics.md (hunk 1 above)
Comment (callum-mcdata) [reply website/docs/docs/building-a-dbt-project/metrics.md]: Actually scratch that, I'm going to change built to define.

File: website/docs/docs/building-a-dbt-project/metrics.md [hunk 2]
```diff
@@ -135,6 +135,12 @@ metrics:
 
 </File>   
 
+:::caution
+
+- You can not defined metrics on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). To define a metric, the materialization must have a represention in the data warehouse.
```
Comment (mirnawong1) [inline website/docs/docs/building-a-dbt-project/metrics.md]: sorry @callum-mcdata , fixed the two typos here!
```suggestion
- You cannot define metrics on [ephemeral models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/materializations#ephemeral). To define a metric, the materialization must have a representation in the data warehouse.
```

**Comment by mirnawong1** [comment]
shipped @callum-mcdata

**Pull request merged**